images/go-ipfs/metadata.yaml

```
@@ -0,0 +1,10 @@
+name: go-ipfs
+image: cgr.dev/chainguard/go-ipfs
+logo: https://storage.googleapis.com/chainguard-academy/logos/go-ipfs.svg
+endoflife: ""
+console_summary: ""
+short_description: Minimalist Wolfi-based image for `go-ipfs`.
+compatibility_notes: ""
+readme_file: README.md
+upstream_url: https://github.com/ipfs/kubo
+keywords:
```

```suggestion
keywords: []
```

This will fix the terraform plan step - the publisher in `tflib/publisher/chainguard.tf` expects a list even if it is empty.